Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -93,6 +93,13 @@ static_assert(sizeof(RenderBlockFlow::MarginInfo) == sizeof(SameSizeAsMarginInfo
 
 namespace BlockStepSizing {
 
+static bool childHasSupportedStyle(const RenderStyle& childStyle)
```

Comment:
When I saw this function I was like this is confusingly vague and then I noticed the namespacing. I think in a long run it's better to have a dedicated helper class in a separate file as opposed to shovel all the things into RenderBlockFlow.cpp. It is already a confusing mix of loosely connected functionalities. TextBoxTrimmer is an example where the feature is limited to block containers yet it's not in RenderBlockFlow and does not require a namespace.

Reply:
Sure! While I somewhat had a preference initially, I don't feel that strongly about it!

Reply:
(Will be moving this in a very soon to come follow up patch)